### PR TITLE
Page providing info about impl Days.

### PR DESCRIPTION
**Pull request by pnkfelix**
~~NOTE: This is a work-in-progress. In particular, the url for the ti.to button (where I hope to garner responses about how many people want to come) is just a TODO right now.~~

Update: Okay I'm done working on this now.

### CODE REVIEW

**Comment by pnkfelix** [comment]
Oh I just realized I should probably saying something about the timing (i.e. what time they should try to show up to Mozilla Paris).

**Comment by pnkfelix** [comment]
Okay, the only thing left is to fill in a proper URL for the `ti.to` button.

I'm **assuming** that having such a button is a good idea, in terms of giving attendees a concrete place to click and say "Okay, I know I definitely responded now."

But I also do not know the way that `ti.to` works. For example, if there is *no* url for people to follow to get to `ti.to`'s sign-up page for `impl Days`, then I should probably just get rid of the button, right?

File: about-impl-days.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: "What is `impl Days`"
+permalink: /about_impl_days/
+---
+
+`impl Days` is a sibling event of RustFest&lt;Paris&gt;, sponsored by
+Mozilla. `impl Days` is an open-ended time for Rust contributors to
```
Comment (Hoverbear) [inline about-impl-days.md]: Also 1aim. cc @yannleretaille

**Review by Hoverbear — APPROVED** [review]
This looks good overall. :) I don't really like having to add the `impl_days_no_br` option but it works!

**Pull request merged**